example/lib/main.dart

```diff
@@ -37,7 +37,8 @@ class _MyHomePageState extends State<MyHomePage> {
         child: Icon(Icons.cleaning_services),
         onPressed: () {
           print("Floating button was pressed.");
-          otpController.clear();
+          otpController.set(['2', '3', '5', '5', '7']);
+          //otpController.clear();
```

Comment:
revert this back to otpController.clear() as this button is meant to clear the otp text.